Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1100,9 +1165,9 @@ namespace Multiplayer
     {
         IConnectionSet& connectionSet = m_networkInterface->GetConnectionSet();
         connectionSet.VisitConnections([readyForEntityUpdates](IConnection& connection)
-        {
-            connection.SendReliablePacket(MultiplayerPackets::ReadyForEntityUpdates(readyForEntityUpdates));
-        });
+            {
```

Comment:
I think your editor is touching everything for some reason and making unrelated whitespace changes, which really clutters up your reviews and obfuscates your actual changes.

Reply:
Reverted whitespace changes. It should be good now.